wrappers/services.go

```diff
@@ -1039,7 +1217,7 @@ func RestartServices(apps []*snap.AppInfo, explicitServices []string,
 // together with their enable/disable status.
 func ServicesEnableState(s *snap.Info, inter Interacter) (map[string]bool, error) {
```

Comment:
is this functions still used, I see it used in snapstate/backend but I'm not sure that the function using itself is used?

Reply:
I put up a PR to remove this https://github.com/snapcore/snapd/pull/13455